src/Illuminate/Foundation/Http/Kernel.php

```diff
@@ -271,7 +271,7 @@ public function hasMiddleware($middleware)
      */
     public function prependMiddleware($middleware)
     {
-        if (array_search($middleware, $this->middleware) === false) {
+        if (!in_array($middleware, $this->middleware)) {
```

Comment:
Strict mode for in_array is strongly recommended. In this case it maybe not so important but for example:
```in_array('any string value', [0, 1, 2])``` is ```true```
```in_array('any string value', [0, 1, 2], true)``` is ```false```

Reply:
@fkulakov Strict vs. non strict call isn't in the scope of this PR

Reply:
@fkulakov Good point :+1: I prefer strictness

Is that compatible with `array_search`?

Reply:
Yes, it's compatible: http://php.net/manual/en/function.array-search.php

But I think ```in_array``` is better way in this case because we need just boolean result whereas ```array_search``` returns key for detected value.